packages/gap/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://www.gap-system.org/
 TERMUX_PKG_DESCRIPTION="GAP is a system for computational discrete algebra, with particular emphasis on Computational Group Theory"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=4.11.1
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION=4.12.1
+TERMUX_PKG_REVISION=0
```

Comment:
Simply remove this line when updating to newer version. No need to have it set to zero.

Reply:
Fixed